File: Modules/SMES/MES/SIE.Web.MES/DataBarcode/Commands/DataBarcodeImportCommand.js
//SIE.defineCommand('SIE.Web.MES.DataBarcode.Commands.DataBarcodeImportCommand', {
//    extend: 'SIE.Web.Common.Import.Commands.ImportExcelCommand',
//    meta: { text: "导入", hierarchy: "导入".t(), group: "business", iconCls: "icon-Upload icon-green" }
//});

SIE.defineCommand('SIE.Web.MES.DataBarcode.Commands.DataBarcodeImportCommand', {
    extend: 'SIE.Web.Common.Import.Commands.ImportCommandBase',
    meta: { text: "导入", hierarchy: "导入".t(), group: "business", iconCls: "icon-Upload icon-blue" },
});

File: Modules/SMES/MES/SIE.Web.MES/DataBarcode/Commands/DataBarcodeImportCommand.cs
using SIE.Common.ImportHelper;
using SIE.Web.Common.Import.Commands;
using System;
using System.Collections.Generic;
using System.Data;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SIE.Web.MES.DataBarcode.Commands
{
    internal class DataBarcodeImportCommand : ImportCommandBase
    {
        protected override ImportCompleted GetImportCompleted()
        {
            return (DataRow[] drSuccess, DataRow[] DrFailed) =>
            {
            };
        }

        protected override Type GetImportHandleType()
        {
            return typeof(SIE.MES.DataBarcode.Handles.DataBarcodeImportHandle);
        }
    }
}
